# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/python-cryptography/hotfix-pyo3.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- cryptography-45.0.3/Cargo.toml.vanilla	2025-06-08 13:11:17.212305849 +0200
+++ cryptography-45.0.3/Cargo.toml	2025-06-08 13:11:50.869982690 +0200
@@ -22,7 +22,7 @@
 
 [workspace.dependencies]
 asn1 = { version = "0.21.3", default-features = false }
-pyo3 = { version = "0.25", features = ["abi3"] }
+pyo3 = { version = "0.25", features = ["abi3", "abi3-py39"] }
 pyo3-build-config = { version = "0.25" }
 openssl = "0.10.72"
 openssl-sys = "0.9.108"
